torch_xla/csrc/tensor_methods.cpp

```diff
@@ -1128,6 +1129,29 @@ XLATensorPtr XLATensor::expand(const XLATensorPtr& input,
   return output;
 }
 
+XLATensorPtr XLATensor::expand_symint(const XLATensorPtr& input,
+                                      c10::SymIntArrayRef sym_size) {
+  SymIntElements size_elements = SymIntElements(sym_size);
+  // // Replace -1 concrete int dim with the true shape value
```

Comment:
can we nuke the commented out code?

Reply:
@miladm I will remove these `-1` handling logic for now. If we want to support it we should add corresponding cpp tests. I am still unsure whether pytorch will convert those -1 for us, my guess is no